autodoc-admin-api/data.lua

```diff
@@ -637,6 +737,26 @@ return {
           * `all` means "run on all nodes", meaning both sidecars in a sidecar-to-sidecar
             scenario. This is useful for tracing/logging plugins.
         ]] },
+        protocols = {
+          description = [[
+            A list of the request protocols that will trigger this plugin. The default value is
```

Comment:
The default value should not be written in the description, it is generated from the actual schema (and printed in JSON syntax) — see the generated .md file above